openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -501,53 +529,15 @@ def shim_student_view(view_func, check_logged_in=False):
 
     Keyword Args:
         check_logged_in (boolean): If true, check whether the user successfully
-            authenticated and if not set the status to 403.
+            authenticated and if not set the status to 403. This argument is
+            used to test the shim by skipping this check.
 
     Returns:
         function
 
     """
     @wraps(view_func)
     def _inner(request):  # pylint: disable=missing-docstring
-        # Make a copy of the current POST request to modify.
-        modified_request = request.POST.copy()
-        if isinstance(request, HttpRequest):
-            # Works for an HttpRequest but not a rest_framework.request.Request.
-            set_custom_metric('shim_request_type', 'traditional')
-            request.POST = modified_request
-        else:
-            set_custom_metric('shim_request_type', 'drf')
-            # The request must be a rest_framework.request.Request.
-            request._data = modified_request  # pylint: disable=protected-access
-
-        # The login and registration handlers in student view try to change
-        # the user's enrollment status if these parameters are present.
-        # Since we want the JavaScript client to communicate directly with
-        # the enrollment API, we want to prevent the student views from
-        # updating enrollments.
-        if "enrollment_action" in modified_request:
-            set_custom_metric('shim_del_enrollment_action', modified_request["enrollment_action"])
-            del modified_request["enrollment_action"]
-        if "course_id" in modified_request:
-            set_custom_metric('shim_del_course_id', modified_request["course_id"])
-            del modified_request["course_id"]
```

Comment:
**Reviewer Note**: Metrics showed this section to be old unused code, so I am simply deleting it.